lib/codegen/src/regalloc/reload.rs

```diff
@@ -209,40 +209,14 @@ impl<'a> Context<'a> {
         debug_assert!(self.candidates.is_empty());
         self.find_candidates(inst, constraints);
 
-        // Insert fill instructions before `inst` and replace `cand.value` with the filled value.
-        for cand in self.candidates.iter_mut() {
-            if let Some(reload) = self.reloads.get(cand.value) {
-                cand.value = reload.reg;
-                continue;
-            }
-
-            let reg = self.cur.ins().fill(cand.value);
-            let fill = self.cur.built_inst();
-
-            self.reloads.insert(ReloadedValue {
-                stack: cand.value,
-                reg,
-            });
-            cand.value = reg;
-
-            // Create a live range for the new reload.
```

Comment:
I'm curious why you removed this comment.

Reply:
Oops, meant to copy that over as well.